plugins/media/wowza/lib/model/WowzaMediaServerNode.php

```diff
@@ -51,7 +51,7 @@ public function getLiveWebServiceName()
 		return WowzaMediaServerNode::WEB_SERVICE_LIVE;
 	}
 	
-	public function getPlaybackHost($protocol = 'http', $format = null)
+	public function getPlaybackHost($protocol = 'http', $format = null, $deliveryType = null)
```

Comment:
This is right for VOD but not right for live since for live we build the urls in the baseEntry get request. There we do not initialize a delivery profile before building the urls.